language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/changetype/FixReturnTypeCodeAction.java

```diff
@@ -97,28 +99,43 @@ public List<CodeAction> getDiagBasedCodeActions(Diagnostic diagnostic,
             return Collections.emptyList();
         }
 
+        List<TextEdit> importEdits = new ArrayList<>();
+        List<String> types = new ArrayList<>();
+        boolean returnsClause = funcDef.get().functionSignature().returnTypeDesc().isPresent();
+
+        if (checkExprDiagnostic) {
+            // Add error return type for check expression
+            String error = "error";
+            String optionalError = "error?";
```

Comment:
Do we need these variables since we have single usage for each?

Reply:
+1, removed.